src/osa/scripts/gain_selection.py

```diff
@@ -126,7 +126,11 @@ def get_sbatch_script(
 
 
 def launch_gainsel_for_data_run(date, run, output_dir, r0_dir, log_dir, log_file, tool, simulate=False):
-    
+    """
+    Create the gain selection sbatch script and launch it for a given run. Runs from before 20231205
+    without UCTS or TIB info are directly copied to the final directory. Subruns that do not have 
+    four streams are also directly copied.
```

Comment:
Maybe also add parameters in the docstrings and/or at least type hints in each argument (date: str, run: Table, etc)